_posts/2019-05-08-deploying-a-dockerized-app-on-gcp-gke.md

```diff
@@ -4,7 +4,7 @@ type: guide
 title: Deploying a Dockerized app on GCP and GKE
 image: /assets/img/guides/gke.jpg
 excerpt: Before we can deploy a dockerized app, we need to first create one. For the purposes of this guide we will create a basic Node.js app that responds to requests on port 8080.
-tags: ["gcp", "gke", "docker"]
+tags: ["gcp", "gke", "docker", "kubernetes"]
```

Comment:
This will conflict with https://github.com/gruntwork-io/gruntwork-io.github.io/pull/182. So feel free to revert, and I'll add your fix to my branch.